xl_auth/translations/sv/LC_MESSAGES/messages.po

```diff
@@ -556,3 +555,6 @@ msgstr "Användarinställningar uppdaterade för \"%(username)s\"."
 msgid "Thank you for changing password for \"%(username)s\"."
 msgstr "Lösenord för \"%(username)s\" har ändrats."
 
+#~ msgid "Logged in as %(user)s"
```

Comment:
This `#~ ...` marks lines that are no longer used and can be removed. So lets remove them!